src/source-af-packet.c

```diff
@@ -527,12 +526,14 @@ static TmEcode AFPPeersListAdd(AFPThreadVars *ptv)
                 continue;
             peer->peer = pitem;
             pitem->peer = peer;
-            mtu = GetIfaceMTU(ptv->iface);
-            out_mtu = GetIfaceMTU(ptv->out_iface);
-            if (mtu != out_mtu) {
+
+            LiveDevice *iface = ptv->livedev;
+            LiveDevice *out_iface = LiveGetDevice(ptv->out_iface);
```

Comment:
looks risky, what if we get a NULL ptr?

Reply:
indeed it is. I should check better. Sending new PR